### PR TITLE
update document for system-table-information-schema.md

### PR DESCRIPTION
**Pull request by wjhuang2016**
Signed-off-by: wjhuang2016 <huangwenjun1997@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: system-tables/system-table-information-schema.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ select * from `ANALYZE_STATUS`;
 6 rows in set
 ```
 
+`ANALYZE_STATUS` 表中列的含义如下：
+
+* `TABLE_SCHEMA`: 表所属的数据库的名称。
+* `TABLE_NAME`: 表的名称。
+* `PARTITION_NAME`: 分区表的名称。
+* `JOB_INFO`: 任务信息。
+* `PROCESSED_ROWS`: 进程数量。
```
Comment (djshow832) [inline system-tables/system-table-information-schema.md:41]: Are you sure?

Comment (wjhuang2016) [reply system-tables/system-table-information-schema.md:41]: Not sure, it's a view that doesn't exist in MySQL.

**Comment by TomShawn** [comment]
@djshow832 PTAL again

File: system-tables/system-table-information-schema.md [hunk 2]
```diff
@@ -548,6 +558,27 @@ desc statistics;
 +---------------+---------------+------+------+---------+-------+
 ```
 
+`TIDB_INDEXES` 表中列的含义如下：
```
Comment (djshow832) [inline system-tables/system-table-information-schema.md:561]: Is it in master? I have much fewer fields in this table.

Comment (wjhuang2016) [reply system-tables/system-table-information-schema.md]: Sorry, `TIDB_INDEXES` is wrong here.

File: system-tables/system-table-information-schema.md [hunk 3]
```diff
@@ -548,6 +558,27 @@ desc statistics;
...
+* `NON_UNIQUE`: 如果索引不能包含重复项，则为 `0`;如果可以，则为 `1`。
+* `INDEX_SCHEMA`: 索引所属的数据库的名称。
+* `INDEX_NAME`: 索引的名称。如果索引是主键，那么名称总是 `PRIMARY`。
+* `SEQ_IN_INDEX`: 索引中的列序号，从1开始。
+* `COLUMN_NAME`: 列名。请参见表达式列的说明。
+* `COLLATION`: 列在索引中的排序方式。它可以有值 `A` (升序)、`D` (降序)或 `NULL` (未排序)。
+* `CARDINALITY`: 索引中唯一值的数量的估计。要更新这个数字，运行 `ANALYZE TABLE`。
+* `SUB_PART`: 索引的前缀。如果只对列的部分前缀索引，则为索引字符的数量;如果对整个列进行索引，则为 `NULL`。
+* `PACKED`: TiDB 未使用该字段。这个值总是 `NULL`。
+* `NULLABLE`: 如果列可能包含空值，则值为 `YES`;如果不包含，则值为 `''`。
```
Comment (djshow832) [inline system-tables/system-table-information-schema.md:575]:
```suggestion
* `NULLABLE`: 如果列可能包含空值，则值为 `YES`；如果不包含，则值为 `''`。
```

File: system-tables/system-table-information-schema.md [hunk 4]
```diff
@@ -548,6 +558,27 @@ desc statistics;
...
+* `INDEX_TYPE`: 索引的类型。
+* `COMMENT`: 索引没有在列中描述的信息。
```
Comment (djshow832) [inline system-tables/system-table-information-schema.md:577]: I don't understand. Please rephrase it.

File: system-tables/system-table-information-schema.md [hunk 5]
```diff
@@ -548,6 +558,27 @@ desc statistics;
...
+`STATISTICS` 表中列的含义如下：
+
+* `TABLE_CATALOG`: 包含索引的表所属的目录的名称。这个值总是 `def`。
+* `TABLE_SCHEMA`: 包含索引的表所属的数据库的名称。
+* `TABLE_NAME`: 包含索引的表的名称。
+* `NON_UNIQUE`: 如果索引不能包含重复项，则为 `0`;如果可以，则为 `1`。
+* `INDEX_SCHEMA`: 索引所属的数据库的名称。
+* `INDEX_NAME`: 索引的名称。如果索引是主键，那么名称总是 `PRIMARY`。
+* `SEQ_IN_INDEX`: 索引中的列序号，从1开始。
+* `COLUMN_NAME`: 列名。请参见表达式列的说明。
```
[the remark this answers is not in the document]
Comment (wjhuang2016) [reply system-tables/system-table-information-schema.md:570]: It's in L580, and I think we don't need to link it.

File: system-tables/system-table-information-schema.md [hunk 6]
```diff
@@ -32,6 +32,16 @@ select * from `ANALYZE_STATUS`;
...
 
+`ANALYZE_STATUS` 表中列的含义如下：
+
+* `TABLE_SCHEMA`: 表所属的数据库的名称。
+* `TABLE_NAME`: 表的名称。
+* `PARTITION_NAME`: 分区表的名称。
+* `JOB_INFO`: 任务信息。
+* `PROCESSED_ROWS`: 已经处理的行数。
+* `START_TIME`: 开始时间。
+* `STATE`: 状态。包括 `padding`，`running`，`finished`和`failed`。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:43]:
```suggestion
* `STATE`: 状态。包括 `padding`、`running`、`finished` 和 `failed`。
```

File: system-tables/system-table-information-schema.md [hunk 7]
```diff
@@ -620,6 +630,27 @@ desc statistics;
...
 
+`STATISTICS` 表中列的含义如下：
+
+* `TABLE_CATALOG`: 包含索引的表所属的目录的名称。这个值总是 `def`。
+* `TABLE_SCHEMA`: 包含索引的表所属的数据库的名称。
+* `TABLE_NAME`: 包含索引的表的名称。
+* `NON_UNIQUE`: 如果索引不能包含重复项，则为 `0`；如果可以，则为 `1`。
+* `INDEX_SCHEMA`: 索引所属的数据库的名称。
+* `INDEX_NAME`: 索引的名称。如果索引是主键，那么名称总是 `PRIMARY`。
+* `SEQ_IN_INDEX`: 索引中的列序号，从 1 开始。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:641]:
```suggestion
* `SEQ_IN_INDEX`: 索引中的列序号，从 `1` 开始。
```

File: system-tables/system-table-information-schema.md [hunk 8]
```diff
@@ -620,6 +630,27 @@ desc statistics;
...
+* `COLUMN_NAME`: 列名。请参见表达式列的说明。
+* `COLLATION`: 列在索引中的排序方式。它可以有值 `A` (升序)、`D` (降序)或 `NULL` (未排序)。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:643]:
```suggestion
* `COLLATION`: 列在索引中的排序方式。取值可以是 `A`（升序）、`D`（降序）或 `NULL`（未排序）。
```

File: system-tables/system-table-information-schema.md [hunk 9]
```diff
@@ -620,6 +630,27 @@ desc statistics;
...
+* `CARDINALITY`: 索引中唯一值的数量的估计。要更新这个数字，运行 `ANALYZE TABLE`。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:644]:
```suggestion
* `CARDINALITY`: 索引中唯一值的数量的估计。要更新这个数字，执行 `ANALYZE TABLE`。
```

File: system-tables/system-table-information-schema.md [hunk 10]
```diff
@@ -620,6 +630,27 @@ desc statistics;
...
+* `SUB_PART`: 索引的前缀。如果只对列的部分前缀索引，则为索引字符的数量；如果对整个列进行索引，则为 `NULL`。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:645]:
```suggestion
* `SUB_PART`: 索引的前缀。如果只对列的部分前缀进行索引，则为索引字符的数量；如果对整个列进行索引，则为 `NULL`。
```

File: system-tables/system-table-information-schema.md [hunk 11]
```diff
@@ -620,6 +630,27 @@ desc statistics;
...
+* `PACKED`: TiDB 未使用该字段。这个值总是 `NULL`。
+* `NULLABLE`: 如果列可能包含空值，则值为 `YES`；如果不包含，则值为 `''`。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:647]:
```suggestion
* `NULLABLE`: 如果列可能包含 `NULL` 值，则值为 `YES`；如果不包含，则值为 `''`。
```

File: system-tables/system-table-information-schema.md [hunk 12]
```diff
@@ -783,10 +814,13 @@ CONSTRAINT_CATALOG: def
...
-其中：
+`TABLE_CONSTRAINTS` 表中列的含义如下：
 
-* `CONSTRAINT_TYPE` 的取值可以是 `UNIQUE`，`PRIMARY KEY`，或者 `FOREIGN KEY`。
-* `UNIQUE` 和 `PRIMARY KEY` 信息与 `SHOW INDEX` 语句的执行结果类似。
+* `CONSTRAINT_CATALOG`: 约束所属的目录的名称。这个值总是 `def`。
+* `CONSTRAINT_SCHEMA`: 约束所属的数据库的名称。
+* `CONSTRAINT_NAME`: 约束的名称。
+* `TABLE_NAME`: 表的名称。
+* `CONSTRAINT_TYPE`: 约束的类型。取值可以是 `UNIQUE`，`PRIMARY KEY`，或者 `FOREIGN KEY`。`UNIQUE` 和 `PRIMARY KEY` 信息与 `SHOW INDEX` 语句的执行结果类似。
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:823]:
```suggestion
* `CONSTRAINT_TYPE`: 约束的类型。取值可以是 `UNIQUE`、`PRIMARY KEY` 或者 `FOREIGN KEY`。`UNIQUE` 和 `PRIMARY KEY` 信息与 `SHOW INDEX` 语句的执行结果类似。
```

**Comment by TomShawn** [comment]
LGTM

File: system-tables/system-table-information-schema.md [hunk 13]
```diff
@@ -31,6 +31,16 @@ select * from `ANALYZE_STATUS`;
...
 
+`ANALYZE_STATUS` 表中列的含义如下：
+
+* `TABLE_SCHEMA`: 表所属的数据库的名称。
+* `TABLE_NAME`: 表的名称。
+* `PARTITION_NAME`: 分区表的名称。
+* `JOB_INFO`: 任务信息。
+* `PROCESSED_ROWS`: 已经处理的行数。
+* `START_TIME`: 开始时间。
+* `STATE`: 状态。包括 `padding`、`running`、`finished` 和 `failed`。
```
Comment (yikeke) [inline system-tables/system-table-information-schema.md:42]: padding 还是 pending？ @wjhuang2016

Comment (yikeke) [reply system-tables/system-table-information-schema.md:42]: Confirmed that pending is correct. Fixed in https://github.com/pingcap/docs-cn/pull/4109/files#diff-304d268879d55212fc021e3b9a605b29R60 and https://github.com/pingcap/docs/pull/3470/files#diff-304d268879d55212fc021e3b9a605b29R60

**Pull request merged**